### PR TITLE
Infra: Gradle init

### PR DESCRIPTION
**Pull request by ajantha-bhat**
- execute `gradle init`
- updated the headers

pre-requisite for #2

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Linux start script should use lf
+/gradlew        text eol=lf
```
Comment (XN137) [inline .gitattributes:5]: nit: wouldnt this be the default?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+networkTimeout=10000
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```
Comment (XN137) [inline gradle/wrapper/gradle-wrapper.properties:6]: in other nessie repos we seem to:

a) also commit `wrapper.jar` https://github.com/projectnessie/nessie/tree/main/gradle/wrapper

b) also set `distributionSha256Sum` https://github.com/projectnessie/nessie/blob/main/gradle/wrapper/gradle-wrapper.properties

File: build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2022 Dremio
```
Comment (XN137) [inline build.gradle.kts:2]: 2023 ? :P

**Review by XN137 — APPROVED** [review]
+1 after making the gradle wrapper stuff like in other repos

wondering though whether this could just be part of the full "add gradle" commit

File: .gitattributes (hunk 1 above)
Comment (ajantha-bhat) [reply .gitattributes:5]: Auto-generated by `gradle init`

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionSha256Sum=312eb12875e1747e05c2f81a4789902d7e4ec5defbd1eefeaccc08acf096505d
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
+networkTimeout=10000
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```
Comment (XN137) [inline gradle/wrapper/gradle-wrapper.properties:7]: nit: missing newline at EOF

File: build.gradle.kts (hunk 3 above)
Comment (ajantha-bhat) [reply build.gradle.kts]: oh yeah. Happy new year ;)

**Review by XN137 — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (ajantha-bhat) [reply gradle/wrapper/gradle-wrapper.properties]: done. Good observation.

File: gradle/wrapper/gradle-wrapper.properties (hunk 4 above)
Comment (ajantha-bhat) [reply gradle/wrapper/gradle-wrapper.properties]: done

**Review by XN137 — APPROVED** [review]

**Pull request merged**